src/runtime-rs/crates/resource/src/share_fs/share_virtio_fs_standalone.rs

```diff
@@ -68,12 +70,14 @@ impl ShareVirtioFsStandalone {
         ensure_dir_exist(&source_path)?;
 
         let mut args: Vec<String> = vec![
-            String::from("-f"),
-            format!("--socket-path={}", sock_path),
-            String::from("-o"),
-            format!("source={}", source_path.to_str().unwrap()),
-            String::from("-o"),
-            format!("cache={}", self.config.virtio_fs_cache),
+            String::from("--socket-path"),
+            String::from(sock_path),
+            String::from("--shared-dir"),
+            String::from(source_path.to_str().unwrap()),
```

Comment:
I can see this was carried over from the original code, but we can avoid the `unwrap()` by using `?` instead as this function returns a `Result`.

Related: https://github.com/kata-containers/kata-containers/blob/main/docs/code-pr-advice.md#rust-language-specifics

Reply:
sure thing

Reply:
done